utils/visualize.py

```diff
@@ -0,0 +1,94 @@
+import io
+from typing import List, Union
+
+import ipywidgets as widgets
+from ase.build import make_supercell
+from ase.io import write
+from IPython.display import display
+from mat3ra.made.material import Material
+from mat3ra.made.tools.convert import to_ase
+from mat3ra.utils.array import convert_to_array_if_not
+
+
+def get_material_visualization_html(material: Material, title: str, rotation="0x,0y,0z", repetitions=[1, 1, 1]):
+    """
+    Returns an image of the material structure with the specified title.
+
+    Args:
+        material (Material): Material object to visualize.
+        title (str): Title of the image.
+        rotation (str): Rotation of the image.
```

Comment:
Add "Rotation of the image, in degrees around the x, y, and z axes (e.g., "-90x,90y,0z")."